File: qz/u2/llumiquinga/ProjectParis_(Llumiquinga&Mera)/RentCar/ec/edu/espe/RentCar/model/customer.py
class Customer:
    def __init__(self, id, name, license_number):
        self.id = id
        self.name = name
        self.license_number = license_number

    def addCustomer(self):
        return {
            "id": self.id,
            "name": self.name,
            "license_number": self.license_number
        }

File: ws/tufiño/u2/ws31 API/models/customer.js
const moongoose = require('mongoose');

const customerSchema = new moongoose.Schema(
    {
        id: { type: Number},
        name: { type: String},
        age: { type: Number},
        moneySpent: { type: Number}
    },
    { collection: 'Customers' }
);

module.exports = moongoose.model('Customer', customerSchema);
